docs/pages/database-access/guides/dynamodb.mdx

```diff
@@ -0,0 +1,298 @@
+---
+title: Database Access with AWS DynamoDB
+description: How to access AWS DynamoDB with Teleport.
+---
+
+<Admonition type="note" title="Important">
```

Comment:
I think we should move this guide to the Application Access section and remove this admonition. This way, we can keep all guides related to a single Teleport resource service in the same place for better discoverability (and also to illustrate all of the capabilities of Application Access).

Reply:
Totally agree this guide is not very coherent with other database guides and it feels like it belongs to Application Access. I believe the original reason to have this guide is customers didn't know Teleport supports `DynamoDB`. If they have been reading the application guide on AWS, they probably can figure this out already. I feel it's more natural for ppl look for a `DynamoDB` guide in `Database Access` section.

I am up for any suggestions to finalize this guide.

@ptgott @r0mant

Reply:
Yeah I think it belongs among the database access guides.

Reply:
That reasoning makes sense!